### PR TITLE
Add the po4a project

### PR DESCRIPTION
**Pull request by mquinson**
This is a follow up of #1018.

### CODE REVIEW

File: _data/projects/po4a.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: po4a
+desc: Maintain the translations of your documentation with ease (PO for anything)
+site: https://po4a.org/
+tags:
+- perl
+- localization
+- internationalization
+upforgrabs:
+  name: beginner friendly
+  link: https://github.com/mquinson/po4a/issues?q=is%3Aissue+is%3Aopen+label%3A%22beginner+friendly%22
```
Comment (ritwik12) [inline _data/projects/po4a.yml:10]: @mquinson Please change this to "https://github.com/mquinson/po4a/labels/beginner%20friendly"

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by mquinson** [comment]
Hello,

I used the online editor to create the PR, without forking the repo. Unfortunately, I cannot edit the file, I can only create another PR. It seems more efficient to me if you merge this request so that I can edit the resulting file instead of restarting from scratch in a third PR.

But I'll create another PR if you prefer me to.

Thanks for your time, Mt.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@mquinson That's fine. thanks for letting me know. I am making the correction for you after merging this.

**Comment by mquinson** [comment]
It seems that you can see when people are using this feature because the PR is marked as coming from `unknown repository` at the top of the page. 

Thanks for your understanding.

**Comment by ritwik12** [comment]
@mquinson Your welcome, but whenever we use online editor then github creates a fork for that itself and all the changes made are stored in that fork. Why not here?

**Pull request merged**